README.md

```diff
@@ -1,76 +1,44 @@
-# LocalLaunch
+# VersaInit
 
-LocalLaunch is a CLI tool that reads a YAML slice of code inside a README.md file and executes the commands specified in the up or down section of the YAML file.
+VersaInit stands for versatile initialization tool. It is a tool written in Go that can help you to automatically bootstrap a project.
 
-## Installation
-To install LocalLaunch, please follow the steps below:
-
-1. Clone this repository to your local machine:
-
-```bash
-<!-- aaa -->
-git clone https://github.com/example/repo.git
-```
-2. Navigate to the cloned directory:
-
-
-```bash
-cd repo
-```
-3. Build the executable:
+## Building
 
+To build the project, you need to have Go installed. Then, you can run the following command:
 
 ```bash
 go build
 ```
-4. Add the executable to your PATH:
 
-
-```bash
-export PATH=$PATH:/path/to/repo
-```
 ## Usage
-To use LocalLaunch, you need to specify the path to your README.md file and the command you want to run. LocalLaunch supports two commands: up and down.
 
-Syntax
+To use the tool, you need to have a configuration file. The configuration file is a YAML file that contains the languages, their corresnponding patterns, and the commands to run for each language. Here is an example of a configuration file:
+
+```yaml
+languages:
+  python:
+    start: "pdm install && pdm start"
+    build: "pdm build"
+    extensions:
+      - "py"
+    special_patterns:
+      - "setup.cfg"
+      - "setup.py"
+      - "pyproject.toml"
+  java:
+    start: gradle bootRun
+    build: gradle build -x check -x test
+    extensions:
+      - "java"
+    special_patterns:
+      - "build.gradle"
+      - "pom.xml"
+```
 
+Then, use `cd` to navigate to the directory of your project and run VersaInit. Below is an example of starting the program with VersaInit:
 
 ```bash
-lol [README.md] [command]
+vinit -c versainit.yaml start
```

Comment:
Do I need to have this configuration file in every repository? (I suppose not, but just to be more clear in the description and examples here). Is that possible to explain here?

Reply:
The documentation will be worked on afterwards. All these will be explained.